src/adic_space.lean

```diff
@@ -308,7 +403,7 @@ end
 -/
 
 -- getting sick of these crappy proofs
-def 𝒞.map_id {X : Type*} [topological_space X] (F : 𝒞 X) : 𝒞.map F F :=
+def 𝒞.map_id {X : Type u} [topological_space X] (F : 𝒞 X) : 𝒞.map F F :=
```

Comment:
maybe it's time to get rid of this crappy comment.

Reply:
Should be gone now.